docs/release-notes/release-notes-1.7.0.md

```diff
@@ -0,0 +1,1434 @@
+# dcrd v1.7.0-rc1
+
+This is a new major release of dcrd.  Some of the key highlights are:
+
+* Four new consensus vote agendas which allow stakeholders to decide whether or
+  not to activate support for the following:
+  * Reverting the Treasury maximum expenditure policy
+  * Enforcing explicit version upgrades
+  * Support for automatic ticket revocations for missed votes
+  * Changing the Proof-of-Work and Proof-of-Stake subsidy split from 60%/30% to 10%/80%
+* Major performance enhancements to unspent transaction output handling
+* Faster cryptographic signature validation
+* Significant improvements to network synchronization
+* Support for a configurable assumed valid block
+* Block index memory usage reduction
+* Asynchronous indexing
+* Version 1 block filters removal
+* Various updates to the RPC server:
+  * Additional per-connection read limits
+  * A more strict cross origin request policy
+  * A new alternative client authentication mechanism based on TLS certificates
+  * Availability of the scripting language version for transaction outputs
+  * Various other RPC modifications, additions, and removals
```

Comment:
Suggest removing this line. 

```
Some of the key highlights are:

...

- Various other RPC modifications, additions, and removals
```

"Various modifications" is not a highlight

Reply:
Hmm, I'm not sure I agree here.  There are several modifications to the results of various RPCs that RPC clients need to be aware of, hence this is highlighting that so they need to look further into them.  Calling each one out individually in the highlights doesn't seem appropriate given the nature of it being a short summary.

Do you have any suggestions for a better way to word that in a succinct manner?

Reply:
Fair point. Leave it as-is then, or maybe go with `Various other *important* RPC modifications, additions, and removals`?

Reply:
After thinking on it a bit, how about just changing the `RPC` to `API` since it's already under a bullet point regarding updates to the RPC server?

Reply:
Yeah good idea.

Reply:
Updated this to:

> Several other notable updates, additions, and removals related to the JSON-RPC API